src/state/SceneDrawerState.tsx

```diff
@@ -8,14 +9,18 @@ import { useState } from '../types/reactHooks'
 import { makeUseContextValue } from '../util/makeUseContextValue'
 import { useSceneScrollContext } from './SceneScrollState'
 
+const SCROLL_DISTANCE = Dimensions.get('window').height / 10
```

Comment:
The window size can change when a tablet rotates. I guess this is fine, since this is just a fudge factor. We could even consider using `theme.rem`, or even a constant `dp` value.

Reply:
Maybe go with a constant (e.g. `theme.rem(3)`) or try leveraging the menu tab height again now that I have those constants in place.